ClimateWatch-Vizzuality/indonesia-platform
# Feature/integrate data uploader

- added Devise
- integrated ActiveAdmin
- configured ActiveStorage
- integrated cw_data_uploader
- configured data uploader for historical emissions, locations and location members datasets

`bundle exec rake db:migrate`
`bundle exec rake db:seeds`
`bundle exec rake db:admin_boilerplate:create`

To test:
- make sure you have S3 access & CW_FILES_PREFIX configured in `.env`
- make sure redis is running
- `bundle exec sidekiq`
- go to `/admin`

Please note for future development: the instructions regarding the data uploader gem are [here](https://github.com/ClimateWatch-Vizzuality/climate-watch-gems/tree/master/data_uploader). Whenever you modify the config/data_uploader.yml file, you'll need to run 2 rake tasks as explained in the instructions.

These changes are fed back to the boilerplate repo [here](https://github.com/ClimateWatch-Vizzuality/project-starter-kit/pull/6).
